database-plugins/src/main/java/io/cdap/plugin/DBRecord.java

```diff
@@ -72,6 +73,7 @@
   public DBRecord(StructuredRecord record, int [] columnTypes) {
     this.record = record;
     this.columnTypes = columnTypes;
+    this.bytesWritten = 0;
```

Comment:
No need. Default value of long field is already 0L.

Reply:
removed.